src/routes/products/orders.js

```diff
@@ -0,0 +1,70 @@
+const path = require("path");
+const fs = require("fs");
+
+
+const getOrder = (request, response) => {
+    const productsFile = path.join(__dirname, '../../../', '/src/db/products', '/all-products.json');
+    const productsFileData = fs.readFileSync(productsFile);
+    const products = JSON.parse(productsFileData);
+
+    const usersFile = path.join(__dirname, '../../../', '/src/db/users', '/all-users.json');
+    const usersFileData = fs.readFileSync(usersFile);
+    const users = JSON.parse(usersFileData);
+    let responseObj = {};
+
+    console.log(arrNormal(request.body.products).length);
+
+    if (getElementsById(request.body.products).length !== arrNormal(request.body.products).length) {
+        responseObj = {
+            'status': 'failed',
+            'order': null
+        }
+    } else {
+        responseObj = {
+            "status": "success",
+            "order": {
+                "id": "id",
+                "user": request.body.user,
+                "products": request.body.products,
+                "deliveryType": request.body.deliveryType,
+                "deliveryAdress": request.body.deliveryAdress
+            }
+        };
+    }
+
+
+    response.writeHead(200, { "Content-Type": "application/json" });
+    response.write(JSON.stringify(responseObj));
+    response.end();
+
+
+
+};
+
+function arrNormal(arr) {
+    let idArr = arr.replace('[', '' || ']', '' || ', ', ',' || ', ', ',');
```

Comment:
remove this function

do this stuff in different way 

think about how to get all ids from array